test/perf/ParseBenchmarks.py

```diff
@@ -1,44 +1,216 @@
+# SPDX-License-Identifier: Apache-2.0
+
+######################################################################################################
+# There are four possible flags to add to this script:
+#
+# --run <Op>: Compute performance benchmarks for specified Op, and write to output file
+#
+# --runall: Compute performance benchmarks for all Ops in OpsWithPerformanceBenchmarks array
+#
+# --readrun <File Op>: Compute performance benchmarks for specified Op, and compare with benchmarks
+# already written to specifiedfile (File must contain same Op)
+#
+# --compare <Op1 Op2>: Compare performance benchmarks written to file for both specified files (each
+# should contain same Op)

```

Comment:
Great, going in the right direction! Can you add an option that create an error if relative error is above X (where X is given by the option). Thanks

Reply:
Sure, on this now. Should this be a multiplier to be applied on each metric? Or should it be for a specific metric (CPU time, etc.)?